eureka/S3_data_reduction/miri.py

```diff
@@ -57,7 +52,7 @@ def read(filename, data, meta):
     if meta.firstFile:
         print('  WARNING: The wavelength for the simulated MIRI data are currently hardcoded '
               'because they are not in the .fits files themselves')
-    data.wave = np.tile(wave_MIRI(filename),(data.data.shape[2],1))[:,::-1]    # hdulist['WAVELENGTH', 1].data
+    data.wave = np.tile(wave_MIRI_hardcoded(),(data.data.shape[2],1))[:,::-1]    # hdulist['WAVELENGTH', 1].data
```

Comment:
Could you add a check to see if `np.all(hdulist['WAVELENGTH`, 1].data == 0)`, and do this hardcoded method and print the warning if so or just set `data.wave = hdulist['WAVELENGTH`, 1].data` and don't print the warning if it is non-zeros? That would help us in the transition period between simulated and real data.

While you're at it, you could add a similar test for the data.time array below where you check if `len(int_times['int_mid_BJD_TDB']) == 0` (True for simulated data, False for real data).

Reply:
I agree. Sure, I'll do that!